tools/contractdeployer/contract_deployer.go

```diff
@@ -203,25 +206,22 @@ func signAndSendTxWithReceipt(wallet wallet.Wallet, client ethadapter.EthClient,
 	}
 
 	var start time.Time
-	var receipt *types.Receipt
-	var contractAddr *common.Address
 	for start = time.Now(); time.Since(start) < timeoutWait; time.Sleep(retryInterval) {
-		receipt, err = client.TransactionReceipt(signedTx.Hash())
+		receipt, err := client.TransactionReceipt(signedTx.Hash())
+		if err != nil {
+			log.Info(err.Error())
```

Comment:
Maybe add something to this like `"Attempt failed, retrying... %w"` so if anyone does look at these logs they don't get too worried by the semi-spam

Reply:
Left in by accident. Removed.